app/src/main/java/com/palone/planahead/data/database/alert/Alert.kt

```diff
@@ -12,17 +12,17 @@ import kotlinx.parcelize.Parcelize
 @Entity(
     foreignKeys = [ForeignKey(
         entity = Task::class,
-        parentColumns = arrayOf("task_id"),
-        childColumns = arrayOf("task_id"),
+        parentColumns = arrayOf("taskId"),
+        childColumns = arrayOf("taskId"),
         onDelete = ForeignKey.SET_NULL
```

Comment:
I think we could have a CASCADE here. An alert has no meaning without a task attached to it.